Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2021"
 [target.'cfg(target_os = "linux")'.dependencies]
 landlock = { git = "https://github.com/phylum-dev/rust-landlock" }
 seccompiler = "0.2.0"
-libc = "0.2.132"
+libc = "0.2.134"
```

Comment:
Technically we don't need to change this, right? Our crate doesn't actually require a libc bump?

Reply:
Well, it would probably resolve to `0.2.134` just the same as it's a dependency of `rust-landlock`. We should make this clear by either having `0.2.134` or `0.2`, as having `0.2.132` might lead someone to think there's a reason we need that specific version number.

Reply:
You should **never** have a dependency without a minor version, because it does not properly represent your crate's dependency on that type.

There is a reason why we specified `0.2.132` specifically and that's the reason why any other library or crate usually specify a specific version: It's the lowest tested dependency version that is guaranteed to work. That's why it's there and that's why we should keep it this way.

This does not mean it's the lowest dependency version that works for this crate and all its dependencies, the two are **completely** unrelated since that's handled by `cargo`. The `0.2.132` version here indicates that this is the lowest tested version required for **our** code.

Reply:
Yes, but we know we haven't _actually_ tested the current code against `0.2.132` as `0.2.134` gets brought in at a minimum due to `rust-landlock` depending on it for the newest addition. So that feels misleading. Either way, the overall outcome is the same, so I've reverted this.

Reply:
There have been no changes to `libc`, thus we don't need to change the version. We haven't tested all of the code against `0.2.132`, correct, but we have tested that all the dependencies we have on `libc` work with `0.2.132`. If the latest code does **not** work with `0.2.132` anymore, that would only be possible by libc introducing a breaking change while bumping patch version.